security/security-design/bastion-session-script/bastion-session.sh

```diff
@@ -0,0 +1,499 @@
+#!/usr/bin/env bash
```

Comment:
Please move this file to a subfolder called /files

@oheimburger code review please